hooks/deploy-product-onexit.sh

```diff
@@ -100,4 +100,68 @@ else
     echo "INFO Successfully deployed manifests for v1.32"
 fi
 
+echo "INFO Checking current k8s-primary-cni value in BSS"
+CNI_VALUE=$(cray bss bootparameters list --hosts Global --format json | jq -r '.[]."cloud-init"."meta-data"."k8s-primary-cni"')
+
+if [[ "$CNI_VALUE" == "cilium" ]]; then
+    echo "INFO k8s-primary-cni is already set to 'cilium'. Skipping migration workflow."
+else
+    echo "INFO k8s-primary-cni is '$CNI_VALUE'. Proceeding with migration workflow."
+
+    echo "INFO Generating Cilium workflow manifest"
+
+    if [[ ! -f /usr/share/doc/csm/workflows/cilium/generateCiliumLiveMigration.py ]]; then
+        echo "ERROR Missing file: generateCiliumLiveMigration.py"
+        exit 1
+    fi
+
+    /usr/share/doc/csm/workflows/cilium/generateCiliumLiveMigration.py
+    if [[ $? -ne 0 ]]; then
```

Comment:
This can be combined to a single line:
```suggestion
    if ! /usr/share/doc/csm/workflows/cilium/generateCiliumLiveMigration.py; then
```
I realize all the code above uses that pattern of running the command and doing a separate step for checking the exit status, so if you'd rather just stick with this pattern (which is not recommended by `shellcheck`), then that's okay.